src/components/Footer/Footer.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <footer>
+    <p><b>Data Source:</b> <a href="https://www.iqair.com/us/air-pollution-data-api" target="_blank">IQAir Air Quality API</a></p>
+    <p><b>Built By:</b> Tashia Davis, Alia Peterson, and Allison Dietz. <a href="https://github.com/alia-peterson/ActivAir" target="_blank">View Repo on GitHub</a></p>
+  </footer>
+</template>
+
+<script>
+export default {
+  name: 'Footer'
+}
+</script>
+
+<style src='./Footer.css'></style>
```

Comment:
This looks great from what I saw on your screen share earlier.
I see the import / exports where I'd expect, but am currently getting this error in trying to view locally: 
![ECCC2EF3-5354-4B39-AF00-48F1348E73B5_4_5005_c](https://user-images.githubusercontent.com/64617223/109434475-cd562500-79d2-11eb-8022-123fce713826.jpeg)